library/lib.mli

```diff
@@ -76,7 +70,6 @@ val make_kn : Id.t -> KerName.t
 
 (** Are we inside an opened section *)
 val sections_are_opened : unit -> bool
-[@@ocaml.deprecated "Use Global.sections_are_opened"]
```

Comment:
Why undeprecate? Is the Global one broken?

Reply:
The Global one requires a full environment, so it is not well suited to components which need to know whether they are in a section at parsing time (and such components exist).